media-libs/libgsf/libgsf-1.14.47.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY=" The G Structured File Library"
+DESCRIPTION=" The project aims to provide an efficient extensible i/o abstraction for \
+dealing with different structured file formats."
+HOMEPAGE="https://github.com/GNOME/libgsf"
+COPYRIGHT="2002-2020 Jody Goldberg"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/GNOME/libgsf/archive/LIBGSF_1_14_47.tar.gz"
+SOURCE_FILENAME="libgsf-$portVersion.tar.gz"
+SOURCE_DIR="libgsf-LIBGSF_1_14_47"
+CHECKSUM_SHA256="ca958db8b91113804e1c8ec772adb7cb3e44f81d404cf941330d47f3e25ccbe7"
+PATCHES=""
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="114.0.47"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+       libgsf$secondaryArchSuffix = $portVersion
+       cmd:gsf_office_thumbnailer = $portVersion
+       cmd:gsf_vba_dump = $portVersion
+       cmd:gsf = $portVersion
+       lib:libgsf$secondaryArchSuffix = $libVersionCompat
+       lib:libgsf_1$secondaryArchSuffix = $libVersionCompat
+       devel:libgsf_1$secondaryArchSuffix = $libVersionCompat
+       "
+
+
+REQUIRES="
+       haiku$secondaryArchSuffix
+       lib:libgio_2.0$secondaryArchSuffix
+       lib:libglib_2.0$secondaryArchSuffix
+       lib:libgobject_2.0$secondaryArchSuffix
+       lib:libintl$secondaryArchSuffix
+       lib:libxml2$secondaryArchSuffix
+       lib:libz$secondaryArchSuffix
+       "
+
+BUILD_REQUIRES="
+       haiku${secondaryArchSuffix}_devel
+       automake
+       autoconf
+       gtk_doc
```

Comment:
Shouldn't be needed (or moved to BUILD_PREREQUIRES)
```diff
-       automake
-       autoconf
-       gtk_doc
```
I just noticed you used spaces, don't :) Use tab(s) instead